server.js

```diff
@@ -156,6 +159,8 @@ initViewEngine();
 /**
  * Register global middlewares
  */
+app.use(slashes(false));
+app.use(redirectsMiddleware);
```

Comment:
Lifting the redirects middleware up to avoid initialising sessions / passport etc. before we redirect